solution-base/mongodb/Makefile

```diff
@@ -3,10 +3,10 @@ CHART_DIR:="${ROOT_DIR}/charts"
 
 CHART_REPO:="https://charts.bitnami.com/bitnami"
 CHART_MONGO_REPLICASET_VERSION:="7.8.0"
-CHART_MONGO_SHARDED_VERSION:="5.1.4"
+CHART_MONGO_SHARDED_VERSION:="6.6.7"
 
 PATCH_DIR:="${ROOT_DIR}/patches"
-PATCH_FILES:="$(shell ls -d ${PATCH_DIR}/*)"
+PATCH_FILES := $(shell ls -d ${PATCH_DIR}/* | tr '\n' ' ')
```

Comment:
change not needed

Reply:
actually without this , the patch files that are retrieved cannot be applied one by one resulting in an error ( I believe path is too long) when we try to apply the patches later on , this was also the main reason why I did the change to go for a loop

Reply:
that is weird...
what is the error?

- if the error is really the line of the path/command line, we can just remove the dir from the variable: `PATCH_FILES := $(shell cd ${PATCH_DIR}/ && ls -d * | tr '\n' ' ')` and change path before apply patches: `@git -C ${PATCH_DIR} apply --check ${PATCH_FILES}` and `@git -C ${PATCH_DIR} apply ${PATCH_FILES}`
- you also changing the patch commands to add the `--3way` param, could this have an effect?